src/Transpiler.ts

```diff
@@ -691,6 +691,30 @@ export abstract class LuaTranspiler {
         }
     }
 
+    public transpileStringLiteralExpression(node: ts.Node): string {
```

Comment:
I would suggest refactoring to `escapeString(str: string): string` and just calling this wherever suitable.